OpenAFSLibrary/keywords/path.py

```diff
@@ -234,18 +236,39 @@ def get_fid(self, path):
         fid = "%d.%d.%d" % (int(volume), int(vnode), int(unique))
         return fid
 
+    def _rilookup(self, fid):
+        """ 
+        This is not a keyword method. It uses RPC to query RIDB on file 
+        server.
+        This needs py_openafs installed.
+        Simplest way is to add the dir to $PYTHONPATH environment variable.
+        """
+        rx.rx_Init(0, 0)
```

Comment:
Is it ok to call rx_Init() everytime the keyword is invoked? Normally, a process calls rx_Init() only once.

Reply:
I think not. Andrew would be a better person to ask this question though. I can ask him and update this. We might need to initialise it someplace else then I suppose.

Reply:
it's more conventional to call `rx_Init` just once, but rx handles multiple inits. I thought there was a kind of "setup" and "teardown" step we could run stuff in? that would be ideal, but init'ing every time will still work

Reply:
Ok, i see, yes rx has a static atomic to track if it is running, so it is safe to call again. We can leave this as is.